### PR TITLE
docs: add v2023.1.2 release notes

### PR DESCRIPTION
**Pull request by blocktrron**

### CODE REVIEW

**Review by maurerle — APPROVED** [review]

**Comment by Djfe** [comment]
Please edit release notes only on
https://md.darmstadt.ccc.de/WBpNrtybSNqtHN-ytxo1FA
instead of Github suggestions

**Review by grische — COMMENTED** [review]
Thanks! I think I was inconsistent with the "Fix" vs "Fixed" in the draft. Apart from that, LGTM.

**Comment by blocktrron** [comment]
@Djfe No, please edit them there before they were Pull-Requested.

File: docs/releases/v2023.1.2.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Bugfixes
+--------
+
+- Fixed Raspberry Pi 3 and 4 naming (`#3099 <https://github.com/freifunk-gluon/gluon/issues/3099>`_)
+
+- Fixed inconsistent usage of env variable BROKEN (`#3103 <https://github.com/freifunk-gluon/gluon/issues/3103>`_)
+
+- Fixed gluon-reconfigure failures when no interface role was selected for an interface (`#3095 <https://github.com/freifunk-gluon/gluon/issues/3095>`_)
+
+- Fix unexpected WiFi shutdowns on TP-Link Archer C7 (`#3049 <https://github.com/freifunk-gluon/gluon/issues/3049>`_)
```
Comment (grische) [inline docs/releases/v2023.1.2.rst:21]:
```suggestion
- Fixed unexpected WiFi shutdowns on TP-Link Archer C7 (`#3049 <https://github.com/freifunk-gluon/gluon/issues/3049>`_)
```

File: docs/releases/v2023.1.2.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- Fix unintentional CPU downclocks of ipq40xx devices (`#3049 <https://github.com/freifunk-gluon/gluon/issues/3049>`_)
```
Comment (grische) [inline docs/releases/v2023.1.2.rst:23]:
```suggestion
- Fixed unintentional CPU downclocks of ipq40xx devices (`#3049 <https://github.com/freifunk-gluon/gluon/issues/3049>`_)
```

File: docs/releases/v2023.1.2.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+- Fix bandwidth downstream (ingress) limit (`#3017 <https://github.com/freifunk-gluon/gluon/issues/3017>`_)
```
Comment (grische) [inline docs/releases/v2023.1.2.rst:25]:
```suggestion
- Fixed bandwidth downstream (ingress) limit (`#3017 <https://github.com/freifunk-gluon/gluon/issues/3017>`_)
```

**Pull request merged**